Modules/zoneinfomodule.c

```diff
@@ -0,0 +1,2695 @@
+#include "Python.h"
```

Comment:
I would prefer to rename the file to Modules/_zoneinfo.c. I don't know where this old tradition of adding "module" in the filename comes from, but I don't think that it adds any value :-p Many modules in Modules/ have no "module" in their filename. I prefer when the C file has the same name than its module + ".c". Example: "import _elementtree" => Modules/_elementtree.c.

Reply:
Done.